source/_integrations/adaptive_lighting.markdown

```diff
@@ -0,0 +1,253 @@
+| `entity_id`               |       no | The `entity_id` of the switch with the settings to apply.               |
+| `lights`                  |       no | A light (or list of lights) to apply the settings to.                   |
+| `transition`              |      yes | The number of seconds for the transition.                               |
+| `adapt_brightness`        |      yes | Whether to change the brightness of the light or not.                   |
+| `adapt_color_temp`        |      yes | Whether to adapt the color temperature on supporting lights.            |
+| `adapt_rgb_color`         |      yes | Whether to adapt the color temperature with `rgb_color`.                |
+| `turn_on_lights`          |      yes | Whether to turn on lights that are currently off.                       |
+
+`adaptive_lighting.set_manual_control` can mark (or unmark) whether a light is "manually controlled", meaning that when a light has `manual_control`, the light is not adapted.
+
+| Service data attribute    | Optional | Description                                                                                        |
+|---------------------------|----------|----------------------------------------------------------------------------------------------------|
+| `entity_id`               |       no | The `entity_id` of the switch in which to (un)mark the light as being "manually controlled".       |
+| `lights`                  |       no | A light (or list of lights) to apply the settings to.                                              |
+| `manual_control`          |       no | Whether to mark (true) or unmark (false) the light as "manually controlled".                       |
+
+
+## Automation examples
+
+Reset the `manual_control` status of a light after an hour.
+```yaml
+- alias: "Adaptive lighting: reset manual_control after 1 hour"
+  mode: parallel
+  trigger:
+    platform: event
+    event_type: adaptive_lighting.manual_control
+  variables:
+    light: "{{ trigger.event.data.entity_id }}"
+    switch: "{{ trigger.entity_id }}"
+  action:
+    - delay: "01:00:00"
+    - condition: template
+      value_template: "{% raw %}{{ light in state_attr(switch, 'manual_control') }}{% endraw %}"
+    - service: adaptive_lighting.set_manual_control
+      data:
+        entity_id: "{{ switch }}"
+        lights: "{{ light }}"
+        manual_control: false
+```
+
+Toggle multiple Adaptive Lighting switches to "sleep mode" using an `input_boolean.sleep_mode`.
+```yaml
+- alias: "Adaptive lighting: toggle 'sleep mode'"
+  trigger:
+    - platform: state
+      entity_id: input_boolean.sleep_mode
+    - platform: homeassistant
+      event: start  # in case the states aren't properly restored
+  variables:
+    sleep_mode: "{{ states('input_boolean.sleep_mode') }}"
+  action:
+    service: "switch.turn_{{ sleep_mode }}"
+    entity_id:
+      - switch.adaptive_lighting_sleep_mode_living_room
+      - switch.adaptive_lighting_sleep_mode_bedroom
+```
```

Comment:
````suggestion
{% raw %}
```yaml
- alias: "Adaptive lighting: toggle 'sleep mode'"
  trigger:
    - platform: state
      entity_id: input_boolean.sleep_mode
    - platform: homeassistant
      event: start  # in case the states aren't properly restored
  variables:
    sleep_mode: "{{ states('input_boolean.sleep_mode') }}"
  action:
    service: "switch.turn_{{ sleep_mode }}"
    entity_id:
      - switch.adaptive_lighting_sleep_mode_living_room
      - switch.adaptive_lighting_sleep_mode_bedroom
```
{% endraw %}
````